src/theme.js

```diff
@@ -1,3 +0,0 @@
-import { createMuiTheme } from '@material-ui/core/styles';
-
-export default createMuiTheme();

```

Comment:
We could use this to test the customization of the theme locally. It wont be shipped to users since `yarn build` only ships from files in `src/components/`. Once we have a big chunk of the schema viewer logic implementation done, we could experiment with something other than the default theme. We could probably leave this file untouched as well as keep the `theme={theme}` part in `src/index.jsx`.

Reply:
> Note: It's possible we will remove this after using the PR re: using a different styleguide. I haven't had the time to look at that PR yet 😅

I'll revert back to how things were before. 
We can discuss how things will be after #39 is merged 😄